src/SonarScanner.MSBuild.Tasks/Targets/SonarQube.Integration.targets

```
@@ -168,6 +169,15 @@
     </ItemGroup>
   </Target>
 
+  <Target Name="WriteSolutionTelemetry" Condition=" $(SonarQubeTempPath) != ''">
+    <WriteTelemetry Condition="$(VisualStudioVersion) != ''" Filename="$(SonarTelemetryFilePath)" Key="dotnetenterprise.s4net.params.visualstudioversion.value" Value="$(VisualStudioVersion)" />
+    <WriteTelemetry Condition="$(VisualStudioVersion) != ''" Filename="$(SonarTelemetryFilePath)" Key="dotnetenterprise.s4net.params.visualstudioversion.source" Value="targets" />
```

Here there is no need for source. There is only one possible source this value can come from
```suggestion
    <WriteTelemetry Condition="$(VisualStudioVersion) != ''" Filename="$(SonarTelemetryFilePath)" Key="dotnetenterprise.s4net.params.visualstudioversion" Value="$(VisualStudioVersion)" />
```